cypress/integration/bulk-edit/bulk-edit-landing-page.spec.js

```diff
@@ -4,7 +4,6 @@ import permissions from '../../support/dictionary/permissions';
 import BulkEditSearchPane from '../../support/fragments/bulk-edit/bulk-edit-search-pane';
 import devTeams from '../../support/dictionary/devTeams';
 import users from '../../support/fragments/users/users';
```

Comment:
Please import the fragment starting from the UpperCase letter

Reply:
done